### PR TITLE
Add headers to configuration

### PR DESCRIPTION
**Pull request by madding**

### CODE REVIEW

**Comment by madding** [comment]
@jmattheis I fixed this issue https://github.com/gotify/server/issues/23, please make review.

File: router/router_test.go [hunk 1]
```diff
@@ -73,6 +76,41 @@ func (s *IntegrationSuite) TestHeaderInProd() {
 	assert.Empty(s.T(), res.Header.Get("Access-Control-Allow-Origin"))
 }
 
+func TestHeadersFromConfiguration(t *testing.T) {
+	mode.Set(mode.Prod)
+	var (
```
Comment (jmattheis) [inline router/router_test.go]: This var statement doesn't look really useful could you remove it?

File: router/router_test.go [hunk 2]
```diff
@@ -73,6 +76,41 @@ func (s *IntegrationSuite) TestHeaderInProd() {
...
+		err error
+		db  *test.Database
+	)
+	db = test.NewDBWithDefaultUser(t)
+	assert.Nil(t, err)
```
Comment (jmattheis) [inline router/router_test.go]: err is unset here, it will always be nil. So this check can be removed.

File: README.md [hunk 3]
```diff
@@ -91,6 +94,7 @@ GOTIFY_SERVER_SSL_LETSENCRYPT_ACCEPTTOS=false
 GOTIFY_SERVER_SSL_LETSENCRYPT_CACHE=certs
 # lists are a little weird but do-able (:
 GOTIFY_SERVER_SSL_LETSENCRYPT_HOSTS=- mydomain.tld\n- myotherdomain.tld
+GOTIFY_SERVER_RESPONSEHEADERS="Access-Control-Allow-Origin: \"*\"\nAccess-Control-Allow-Methods: \"GET,POST\""
```
Comment (jmattheis) [inline README.md]:
```suggestion
GOTIFY_SERVER_RESPONSEHEADERS="Access-Control-Allow-Origin: \"*\"\nAccess-Control-Allow-Methods: GET,POST"
```

File: config/config_test.go [hunk 4]
```diff
@@ -48,6 +53,9 @@ server:
     letsencrypt:
       hosts:
         - push.example.tld
+  response-headers:
+    Access-Control-Allow-Origin: "*"
+    Access-Control-Allow-Methods: "GET,POST"
```
Comment (jmattheis) [inline config/config_test.go]:
```suggestion
    Access-Control-Allow-Methods: GET,POST
```

**Comment by jmattheis** [comment]
Could you squash all your commits?

**Pull request merged**